### DISCUSSION TITLE
Raspberry Pi 4 available at german distributor

### DISCUSSION THREAD

**Discussion by nilspupils** [mcguirepr89/BirdNET-Pi#344] [General]
Normally priced Raspi 4 are now available at BerryBase, if you join their "Makers Club" (for free). 
[BerryBase](https://www.berrybase.de/)
(It is a great shop i have used for a long time!)

**Comment by oliverk71** [comment]
Only 2 GB available. :(
But if 2 GB are sufficient for birdnetpi, then I will order one.

**Reply by nilspupils** [reply]
Check for kits, got a 4 GB one last week.

**Comment by snaterendesnader** [comment]
I managed to order a Pi4 1gb at [Reichelt](www.reichelt.de), although it wasn a bit pricier than needed (€49). They still have some kits available though.
